### DISCUSSION TITLE
Github action repo not at latest version

### DISCUSSION THREAD

**Discussion by vn7n24fzkq** [community/community#26028] [Actions]
<p>My repo <a href="https://github.com/vn7n24fzkq/github-profile-summary-cards" rel="nofollow noopener">https://github.com/vn7n24fzkq/github-profile-summary-cards</a> is updated, but on my github readme or other repos can’t use latest commit in action, it seems can’t fetch latest version.</p>
<p>For example:  <a href="https://github.com/vn7n24fzkq/vn7n24fzkq/blob/7c5b44065884565648c9ad244748a27e16be3618/profile-summary-card-output/solarized/profile-details.svg" rel="nofollow noopener">profile-details.svg</a><br>
Should be updated and looks like this one<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/702f0b79e8f2f18f76af0c794ce9e4f75f7a6ded.png" data-download-href="/uploads/short-url/g0qgJ05MnM9xYRpVZmB1WdMkpbv.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/20241522/181094859-15cb9ffd-336f-4eba-954a-70de3d8dd86d.png" alt="image" data-base62-sha1="g0qgJ05MnM9xYRpVZmB1WdMkpbv" width="690" height="201" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/702f0b79e8f2f18f76af0c794ce9e4f75f7a6ded_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">698×204 16.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Here is my workflow.<br>
<a href="https://github.com/vn7n24fzkq/vn7n24fzkq/edit/master/.github/workflows/main.yml" rel="nofollow noopener">my workflow</a></p>
<blockquote>
<p>name: Generate profile summary cards</p>
<p>on:<br>
workflow_dispatch:<br>
push:<br>
branches: [ master ]<br>
pull_request:<br>
branches: [ master ]<br>
schedule: # execute every 3 hours<br>
- cron: ‘* */3 * * *’</p>
<p>jobs:<br>
build:<br>
runs-on: ubuntu-latest</p>
<pre><code>steps:
  - uses: actions/checkout@v2
  - uses: vn7n24fzkq/github-profile-summary-cards@release
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN}}
    with:
      USERNAME: ${{ github.repository_owner }}
</code></pre>
</blockquote>
<p>And I’m sure release branch is on latest commit.</p>
<p>Does anyone get same situation ?</p>

**Comment by NicoNBW** [comment, marked as answer]
<p>Thanks for your feedback.<br>
I checked your code in release branch, and found that the problem may be because you did not modify your code completely. Please check whether the code that should be modified has been updated?<br>
Such as <a href="https://github.com/vn7n24fzkq/github-profile-summary-cards/blob/8cdc681787769475d658a92bb2367d534f570b99/dist/index.js" rel="nofollow noopener">the index.js file</a> :<br>
<img src="https://user-images.githubusercontent.com/46212442/181094860-832eddc0-1754-422c-b0e9-670684afbb92.png" alt="image" data-base62-sha1="9HIdfoWQibUgi6qg4JpxB87L6tU" width="412" height="190"></p>

**Comment by vn7n24fzkq** [comment]
<p>Thank you so much!<br>
You really made my day.</p>
<p>I use <a href="https://github.com/vercel/ncc" rel="nofollow noopener">ncc</a> to compile nodejs module to index.js, and I forgot to recompile when I modify code.</p>
